run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$NODE_ENV" = "ci" ]
+then echo "starting service"
+  SITEROOT=/firearms GA_TAG_ID=UA-72527484-1
```

Comment:
TBH, I am not sure what SITEROOT is for. Does anyone lnow if it's still required? @easternbloc @gxxm @joefitter @bendiggle ???